rke/k8s_version_info.go

```diff
@@ -181,9 +181,9 @@ func loadK8sVersionInfo() map[string]v3.K8sVersionInfo {
 		},
 		// The Calico/Canal template in this version use functions that are only available in RKE v1.0.0 and up
 		// This version includes nodelocal dns only available in RKE v1.0.7 and up
-		"v1.17.6-rancher2-3": {
-			MinRancherVersion: "2.4.6-rc0",
-			MinRKEVersion:     "1.1.4-rc0",
+		"v1.17.9-rancher1-1": {
+			MinRancherVersion: "2.4.4-rc0",
+			MinRKEVersion:     "1.1.2-rc0",
```

Comment:
minRKEversion should be 1.1.4-rc0?

Reply:
For 2.4.4, we use 1.1.2 https://github.com/rancher/rancher/blob/v2.4.4/go.mod#L86